### PR TITLE
Add syntax highlighting themes

### PR DESCRIPTION
**Pull request by ianchanning**
This builds on top of patch-1 PR ( #5 ).

This is my solution for issue #4.

Rather than have separate options I have called the `css` paramter `theme` now and then based off of that generate a css file path and add a new `highlight` directory with `*.theme` files that match the css names.

This means that the interface doesn't need to change.

I've done testing that either setting the `g:nvim_markdown_preview_theme` var or calling `:MarkdownPreview <theme>` still works for all 3 themes.

### CODE REVIEW

**Comment by davidgranstrom** [comment]
@ianchanning Thank you, this looks great! 👍 

This feels pretty much done! But I think we should introduce `g:nvim_markdown_preview_format` to let users set the pandoc `--from/-f` flag as they see fit. I couldn't really spot the difference between `gfm` and `markdown` in my very brief test of this branch, but I will try some more examples.. I'm not sure what the default should be yet, but for backwards compatibility reasons `gfm` seems like the most reasonable candidate? Documentation for this new option should also be added in `doc/nvim-markdown-preview.txt` and possibly also in the README.

I'll leave some minor comments in the review!

File: README.md [hunk 1]
```diff
@@ -72,6 +107,8 @@ The markdown file must be written to disk first. It should be possible to use Pa
 
 ## License
 
+GPL v3
```
Comment (davidgranstrom) [inline README.md:137]: This can be removed since the information is already contained in the license block below.

File: ftplugin/markdown.vim [hunk 2]
```diff
@@ -8,10 +8,10 @@ if exists('b:did_nvim_markdown_preview')
 endif
 let b:did_nvim_markdown_preview = 1
 
-function! s:markdown_preview(css)
+function! s:markdown_preview(theme)
   let default_theme = get(g:, 'nvim_markdown_preview_theme', 'github')
-  let s:stylesheet = a:css != '' ? a:css : default_theme
-  let s:stylesheet = s:stylesheet . '.css'
+  let s:stylesheet = a:theme != '' ? a:theme : default_theme
+  " let s:stylesheet = s:stylesheet . '.css'
```
Comment (davidgranstrom) [inline ftplugin/markdown.vim]: This comment can also be removed.

File: README.md (hunk 1 above)
Comment (ianchanning) [reply README.md:137]: I try to make things super obvious for people so they don't have to read the dense text of the license.

**Comment by davidgranstrom** [comment]
@ianchanning Just to clarify, I can add the option I mentioned earlier (unless you want to) I just wanted to hear your feedback before pushing anything new.

**Comment by ianchanning** [comment]
Yeah that's great - I'd rather you do that as I don't want to mess up the structure of your code. 

On 1 December 2021 07:32:26 CET, "David Granström" ***@***.***> wrote:
***@***.*** Just to clarify, I can add the option I mentioned earlier (unless you want to) I just wanted to hear your feedback before pushing anything new.

**Comment by davidgranstrom** [comment]
@ianchanning Thanks!

**Comment by ianchanning** [comment]
Great, thank you for merging :)

**Pull request merged**